presto-main/src/test/java/com/facebook/presto/cost/TestSemiJoinStatsCalculator.java

```diff
@@ -159,16 +159,23 @@ public void testSemiJoin()
 
         // source stats are unknown
         assertThat(semiJoinStatsCalculator.computeSemiJoin(inputStatistics, inputStatistics, unknown, u))
-                .symbolStatsUnknown(unknown)
-                .symbolStatsUnknown(u)
-                .symbolStatsUnknown(z)
+                .symbolStats(unknown, stats -> stats
+                        .nullsFraction(0)
+                        .distinctValuesCountUnknown()
+                        .unknownRange())
+                .symbolStats(u, stats -> stats.isEqualTo(uStats))
+                .symbolStats(z, stats -> stats.isEqualTo(zStats))
                 .outputRowsCountUnknown();
 
         // filtering stats are unknown
         assertThat(semiJoinStatsCalculator.computeSemiJoin(inputStatistics, inputStatistics, x, unknown))
-                .symbolStatsUnknown(x)
+                .symbolStats(x, stats -> stats
+                        .nullsFraction(0)
+                        .lowValue(xStats.getLowValue())
+                        .highValue(xStats.getHighValue())
+                        .distinctValuesCountUnknown())
                 .symbolStatsUnknown(unknown)
-                .symbolStatsUnknown(z)
+                .symbolStats(z, stats -> stats.isEqualTo(zStats))
```

Comment:
Or `.symbolStats(z, zStats::isEqualTo)`. Or do we have (should we have?) simpler `.symbolStats(z, zStats)`?